pkg/query-service/app/traces/v3/query_builder.go

```diff
@@ -297,12 +303,27 @@ func buildTracesQuery(start, end, step int64, mq *v3.BuilderQuery, tableName str
 		query := fmt.Sprintf(queryTmpl, step, op, filterSubQuery, groupBy, having, orderBy)
 		return query, nil
 	case v3.AggregateOperatorNoOp:
-		// queryTmpl := constants.TracesSQLSelect + "from " + constants.SIGNOZ_TRACE_DBNAME + "." + constants.SIGNOZ_SPAN_INDEX_TABLENAME + " where %s %s"
-		// query := fmt.Sprintf(queryTmpl, spanIndexTableTimeFilter, filterSubQuery)
-		// return query, nil
-		return "", fmt.Errorf("not implemented, part of traces page")
+		var query string
+		if panelType == v3.PanelTypeTrace {
+			withSubQuery := fmt.Sprintf(constants.TracesExplorerViewSQLSelectWithSubQuery, spanIndexTableTimeFilter, filterSubQuery)
+			withSubQuery = addLimitToQuery(withSubQuery, mq.Limit, panelType)
+			if mq.Offset != 0 {
+				withSubQuery = addOffsetToQuery(withSubQuery, mq.Offset)
+			}
+			query = withSubQuery + ") " + constants.TracesExplorerViewSQLSelectQuery
+		} else if panelType == v3.PanelTypeList {
+			if len(mq.SelectColumns) == 0 {
+				return "", fmt.Errorf("select columns cannot be empty for panelType %s", panelType)
+			}
+			selectColumns := getSelectColumns(mq.SelectColumns, keys)
+			queryNoOpTmpl := fmt.Sprintf("SELECT timestamp as timestamp_datetime, spanID, traceID, "+"%s ", selectColumns) + "from " + constants.SIGNOZ_TRACE_DBNAME + "." + constants.SIGNOZ_SPAN_INDEX_TABLENAME + " where %s %s"
```

Comment:
Why alias `timestamp` as `timestamp_datetime` and add an additional branch here https://github.com/SigNoz/signoz/pull/2847/files#diff-a8c15149b48aee5261176aa62fa630bb7bde1c4362583ff95871ba24cec75bf9R4226

Reply:
This is because timestamp is in unix int in logs and in traces it's in time.Time